packages/angular/cli/src/utilities/config.ts

```diff
@@ -21,21 +21,25 @@ function isJsonObject(value: json.JsonValue | undefined): value is json.JsonObje
 function createWorkspaceHost(): workspaces.WorkspaceHost {
   return {
     async readFile(path) {
-      return readFileSync(path, 'utf-8');
+      return fs.readFile(path, 'utf-8');
     },
     async writeFile(path, data) {
-      writeFileSync(path, data);
+      return fs.writeFile(path, data);
```

Comment:
Nit: `WorkspaceHost.writeFile()` is declared to return `Promise<void>`. If `fs.writeFile()` changes in the future, this could introduce a type error or start returning a value at runtime which isn't defined by the type. I suggest dropping the `return`, or updating `WorkspaceHost` to be dependent on `fs`'s real types if that is the behavior we actually want.